module_utils/ansible_nailgun_cement.py

```diff
@@ -48,6 +49,11 @@
 from ansible.module_utils.parsing.convert_bool import boolean
 
 
+class SyncPlan(SyncPlan):
+    # What does Nailgun think here? (The transformation is done in to_json_serializable anyway)
+    create_payload = entity_mixins.EntityCreateMixin.create_payload
```

Comment:
I think `to_json_serializable` was added more recently than `SyncPlan.create_payload`, I assume it just wasn't removed. Why not raise a PR with Nailgun instead?

Reply:
Will do anyway. But i started to dislike waiting for PR's in other projects.

Reply:
https://github.com/SatelliteQE/nailgun/pull/622